index.js

```diff
@@ -1,3 +1,4 @@
 'use strict';
 
+require('setimmediate');
```

Comment:
This `index.js` is main root, the library should be in `dependencies`.
But I don't want to add it to `dependencies`, is it possible to use another main root for build?